src/components/MultiSelect/MultiSelect.mdx

```diff
@@ -0,0 +1,716 @@
+components and its accessibility compliance is derived from what is already
+included in those components.
+
+Accordion:
+
+- Follows [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/components/accordion/)
+  guidelines:
+  - Use the `button` HTML element for screenreaders and keyboards.
+  - Use the `aria-expanded` attribute to indicate whether the `Accordion` is
+    expanded (`true`) or collapsed (`false`).
+  - Use the `aria-controls` attribute to associate the control with the panel.
+- The open and close icons are decorative (`aria-hidden` is `true`).
+- Visible focus goes around full button and full button is clickable.
+
+Checkbox:
+
+The Checkbox component renders `<label>` and `<input type="checkbox" />`
+elements that are associated with each other; the label has a `for` attribute with
+the value of the `input`'s `id` element. If there is helper or invalid text, the
+`<input>` element will have a `aria-describedby` attribute with the value of the
+helper or invalid text.
+
+Resources:
+
+- [Reservoir Accordion Component](../?path=/docs/components-overlays-switchers-accordion--docs#accessibility)
+- [Reservoir Checkbox Component](../?path=/docs/components-form-elements-checkbox--docs#accessibility)
+- [Deque University - Nested interactive controls are not announced by screen readers](https://dequeuniversity.com/rules/axe/4.4/nested-interactive)
+
+## Populating the List Items
+
+The options rendered in the `MultiSelect` component are passed into the
+component's `items` prop. The expected data structure is an array of objects,
+with each item having an `id` and a `name`. You also have the option to include
+a second level of child items. Both parent and child items can optionally use
+the `isDisabled` flag. The format is as follows:
+
+<Source
+  code={`
+<MultiSelect
+  buttonText="MultiSelect"
+  id="multi-select-id"
+  isBlockElement
+  items={[
+    {
+      id: "art",
+      name: "Art",
+    },
+    {
+      id: "architecture",
+      name: "Architecture",
+      isDisabled: true,
+    },
+    {
+      id: 'design',
+      name: 'Design',
+      children: [
+        {
+          id: 'fashion',
+          isDisabled: true,
+          name: 'Fashion'
+        },
+        {
+          id: 'ux',
+          isDisabled: true,
+          name: 'User Experience'
+        },
+        {
+          id: 'tecture',
+          isDisabled: false,
+          name: 'Architecture'
+        },
+        {
+          id: 'home',
+          isDisabled: false,
+          name: 'Home'
+        },
+      ],
+    },
+    ...
+  ]}
+/>
+`}
+  language="jsx"
+/>
+
+## Visible List Items
+
+The `defaultItemsVisible` prop can be used to set the number of list items that
+will be visible when the list is first opened. The default value is `5`.
+
+When there are nested list items, only the parent items will be considered when
+counting the items to display.
+
+**IMPORTANT:** This prop can only be used with `listOverflow="expand"`.
```

Comment:
I like the use of this as it stands out. Perhaps we can consider using the DS Notification to highlight these notes? Building the DS storybook with DS components -- an idea for later but just pointing out that this is readable already.